### PR TITLE
Feature/content overflow

### PR DESCRIPTION
**Pull request by umeshv12**
Feature/content overflow code changes

### CODE REVIEW

**Review by ashishgoyalweb — CHANGES_REQUESTED** [review]
Optimize less file and use font awesome icons instead of third party.  Exclude Sign-in js

**Comment by umeshv12** [comment]
> Optimize less file and use font awesome icons instead of third party. Exclude Sign-in js

Implemented the changes

**Comment by umeshv12** [comment]
- Author mode the client JS execution will be skipped, so the overflow container appears same as normal container
- Arrows padding for mobile screen is fixed

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/js/tiny-slider.js [hunk 1]
```diff
@@ -0,0 +1,3178 @@
+var tns = (function (){
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/js/tiny-slider.js]: @Rajamouli-hcl : this tiny-slider.js should go in clientlib-site and not in component clientlib

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/js.txt [hunk 2]
```diff
@@ -1 +1,3 @@
-js/container.js
\ No newline at end of file
+js/container.js
+js/tiny-slider.js
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/js.txt]: @Rajamouli-hcl : this should be part of clientlib site

**Review by rishabh-behl — COMMENTED** [review]
@Rajamouli-hcl :pl incorporate comments

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/js.txt (hunk 2 above)
Comment (umeshv12) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/js.txt]: Updated as per comments!

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/js/tiny-slider.js (hunk 1 above)
Comment (umeshv12) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/js/tiny-slider.js]: Updated as per comments!

**Review by rishabh-behl — APPROVED** [review]
LGTM

**Pull request merged**